### PR TITLE
Adding CYOA writeup

### PR DESCRIPTION
**Pull request by Ariana1729**
Added a CYOA writeup file with the challenge and chemicals

### CODE REVIEW

**Comment by daniellimws** [comment]
Change challenge.txt to README.md

**Comment by daniellimws** [comment]
Oh oops saw wrongly

File: crossctf-quals2018/CYOA/README.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# Choose Your Own Adventure - CrossCTF Quals 2018 (misc)
+
+> Maybe your question was too long. You *reduced* the length of your question and asked the oracle again: "Where's the flag?"
+> As expected, she had no *reaction* again. You sigh... maybe the *solution* really was in there after all. You close your eyes and started willing yourself back into the enemy's base...
+> Non-standard flag format: <code>[A-Z0-9]+</code>
+
+## Challenge
+We are given the hints in this format:
+
+>01 01 02, Score: 04
```
Comment (daniellimws) [inline crossctf-quals2018/CYOA/README.md:12]: Add newlines for the few lines below

**Review by daniellimws — CHANGES_REQUESTED** [review]

File: crossctf-quals2018/CYOA/README.md (hunk 1 above)
Comment (Ariana1729) [reply crossctf-quals2018/CYOA/README.md:12]: Do you want writeups for RSA 1 and 2 too?

**Comment by Ariana1729** [comment]
Did the change go through or did i not commit the change here?

File: crossctf-quals2018/CYOA/README.md (hunk 1 above)
Comment (daniellimws) [reply crossctf-quals2018/CYOA/README.md:12]: Sure thanks

File: crossctf-quals2018/CYOA/README.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+>6 -- Down
+>
+>9 -- Quit
+
+1-6 are used to navigate the location to collect all the hints.
+
+And we are also given 4 hints to solve the challenge:
+>Hint 1 for CYOA: len(flag) = 9 Think outside the box for this challenge. It's less of a ctf challenge and more of a misc puzzle tbh...
+>
+>Think about how the contents of each room relate to the coordinates and the score.
```
Comment (daniellimws) [inline crossctf-quals2018/CYOA/README.md:31]: Hint 2:

File: crossctf-quals2018/CYOA/README.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+>
+>So now that you can match why each object is in a specific room (with a certain coordinate and score), perhaps think about the scribblings. What's the meaning behind the scribbles?
```
Comment (daniellimws) [inline crossctf-quals2018/CYOA/README.md:33]: Hint 3:

File: crossctf-quals2018/CYOA/README.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+>
+>You are looking for a string of characters that leads to the final flag. Link each object in the rooms to the coordinates and figure out the overarching theme of the puzzle.
```
Comment (daniellimws) [inline crossctf-quals2018/CYOA/README.md:35]: Hint 4:

File: crossctf-quals2018/CYOA/README.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+
+And the words reduced, reaction and solution are italicized, suggesting the topic is chemistry.
+
+## Interpreting the hints
+
+Consider the hint above, urea is one component of urine, with chemical formula CO(NH2)2 -> C1O2N2H4, which corresponds to the coordinates(01 01 02) and the score(04). With this in mind we check all the other 19 hints and get all the chemical formula.
```
Comment (daniellimws) [inline crossctf-quals2018/CYOA/README.md]: Add link to "cyoa challenge.txt" and "cyoa chemicals.txt"

**Review by daniellimws — CHANGES_REQUESTED** [review]

**Comment by daniellimws** [comment]
Yea the changes went through

**Comment by Ariana1729** [comment]
Added the changes

**Comment by daniellimws** [comment]
Er can rename challange to challenge?

**Comment by Ariana1729** [comment]
Oops thanks

**Comment by daniellimws** [comment]
Ok thanks

**Pull request merged**